scipy/sparse/base.py

```diff
@@ -396,11 +396,11 @@ def _sub_sparse(self, other):
         return self.tocsr()._sub_sparse(other)
 
     def _sub_dense(self, other):
-        return self.todense() - other
+        return self.toarray() - other
```

Comment:
This and the `_rsub_dense` change aren't backwards compatible, unfortunately. Let's leave these out of this PR.

Reply:
And the `_divide` changes as well.